kustomize/keycloak/keycloak.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: keycloak
-  namespace: postgres-operator
+  namespace: keycloak
```

Comment:
Can we keep this set to the `postgres-operator` namespace since our examples, etc. will expect `postgres-operator`?